### PR TITLE
Fix selenium issues

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/micromasters/issues/3934

#### What's this PR do?
it fixes selenium issues which are introduce after model changes.

#### How should this be manually tested?
run `./scripts/test/run_snapshot_dashboard_states.sh`

@pdpinch

### CODE REVIEW

File: selenium_tests/management/commands/snapshot_dashboard_states.py [hunk 1]
```diff
@@ -290,7 +290,7 @@ def create_passed_and_offered_course_run(self, grades_frozen, with_certificate):
             final_grade = FinalGrade.objects.filter(user=self.user, course_run__course=course, passed=True).first()
             CourseRunGradingStatus.objects.create(course_run=final_grade.course_run, status='complete')
             if with_certificate:
-                MicromastersCourseCertificate.objects.create(final_grade=final_grade)
+                MicromastersCourseCertificate.objects.create()
```
[the remark this answers is not in the document]
Comment (amir-qayyum-khan) [reply selenium_tests/management/commands/snapshot_dashboard_states.py]: added, @pdpinch this PR is not ready for review yet. I am testing it

**Comment by amir-qayyum-khan** [comment]
@pdpinch it is ready for review.

File: selenium_tests/data_util.py [hunk 2]
```diff
@@ -50,13 +50,13 @@ def create_user_for_login(is_staff=True, username=None):
...
             image_medium=None,
-            **({'user_username': username} if username is not None else {}),
             user__social_auth__extra_data={
                 'access_token': 'fake',
                 'refresh_token': 'fake',
                 'updated_at': later.timestamp(),
                 'expires_in': 3600,
-            }
+            },
+            **({'user__username': username} if username is not None else {}),
```
Comment (pdpinch) [inline selenium_tests/data_util.py]: possibly dumb question: is there a reason this was moved from line 53 to line 59?

Comment (amir-qayyum-khan) [reply selenium_tests/data_util.py]: there was a warning that python 3.6 do not support **() operator in middle.

Comment (amir-qayyum-khan) [reply selenium_tests/data_util.py]: screen shot 2018-04-10 at 7 28 42 pm

Comment (noisecapella) [reply selenium_tests/data_util.py]: That warning says `<` not `+`. We are using Python 3.6 and are not planning on going backward so I think we should move the line back where it was

Comment (amir-qayyum-khan) [reply selenium_tests/data_util.py]: reverted @noisecapella

**Pull request merged**